wallet/src/de/schildbach/wallet/ui/widget/ShortcutsPane.kt

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2020 Dash Core Group.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package de.schildbach.wallet.ui.widget
+
+import android.content.Context
+import android.util.AttributeSet
+import android.util.DisplayMetrics
+import android.view.Gravity
+import android.view.View
+import android.view.ViewGroup
+import android.view.ViewTreeObserver
+import android.widget.LinearLayout
+import androidx.core.view.children
+import de.schildbach.wallet_test.R
+
+class ShortcutsPane(context: Context, attrs: AttributeSet) : LinearLayout(context, attrs), View.OnClickListener {
+
+    val secureNowButton: ShortcutButton by lazy {
+        ShortcutButton(context,
+                R.drawable.ic_shortcut_secure_now,
+                R.string.shortcut_secure_now,
+                this)
+    }
+    val receiveButton: ShortcutButton by lazy {
+        ShortcutButton(context,
+                R.drawable.ic_shortcut_receive,
+                R.string.shortcut_receive,
+                this)
+    }
+    val scanToPayButton: ShortcutButton by lazy {
+        ShortcutButton(context,
+                R.drawable.ic_shortcut_scan_to_pay,
+                R.string.shortcut_scan_to_pay,
+                this)
+    }
+    val payToAddressButton: ShortcutButton by lazy {
+        ShortcutButton(context,
+                R.drawable.ic_shortcut_pay_to_address,
+                R.string.shortcut_pay_to_address,
+                this)
+    }
+    val buySellButton: ShortcutButton by lazy {
+        ShortcutButton(context,
+                R.drawable.ic_shortcut_buy_sell_dash,
+                R.string.shortcut_buy_sell,
+                this)
+    }
+    val importPrivateKey: ShortcutButton by lazy {
+        ShortcutButton(context,
+                R.drawable.ic_shortcut_import_key,
+                R.string.shortcut_import_key,
+                this)
+    }
+    val configButton: ShortcutButton by lazy {
+        ShortcutButton(context,
+                R.drawable.ic_shortcut_add,
+                R.string.shortcut_add_shortcut,
+                this)
+    }
+
+    private var isSmallScreen = resources.displayMetrics.densityDpi <= DisplayMetrics.DENSITY_MEDIUM
```

Comment:
The K5, which is considered a small screen has DENSITY_HIGH

Reply:
However, I see that below a different calculation is made on the K5 and it only shows 3 buttons.  Good Work.